a10_neutron_lbaas/neutron_ext/extensions/a10DeviceInstance.py

```diff
@@ -18,7 +18,18 @@
 from a10_openstack_lib.resources import a10_device_instance
 import a10_openstack_lib.resources.validators as a10_validators
 
-from neutron.api import extensions
+# Get the ExtensionDescriptor class from <= Mitaka (? I think)
+try:
+    from neutron.api.extensions import ExtensionDescriptor
+except AttributeError:
+    pass
```

Comment:
import it from neutron_lib here instead of passing. If they both fail it should be an exception.